.travis.yml

```diff
@@ -33,8 +33,36 @@ matrix:
       env: TARGET=test-node
     - node_js: '7'
       env: TARGET=lint
+    # phantomjs
     - node_js: '7'
-      env: TARGET=test-browser S3=1
+      env: TARGET=test-browser
```

Comment:
I mentioned this in the original issue, but how can we ensure we don't open too many concurrent sessions if all of these jobs one concurrently on Travis?